server/src/test/java/io/crate/execution/ddl/TransportSchemaUpdateActionTest.java

```diff
@@ -49,59 +49,6 @@
 
 public class TransportSchemaUpdateActionTest extends CrateDummyClusterServiceUnitTest {
 
-    @Test
-    public void testTemplateMappingUpdateFailsIfTypeIsDifferent() throws Exception {
-        SQLExecutor e = SQLExecutor.builder(clusterService)
-            .addPartitionedTable(
-                "create table t (p int) partitioned by (p)"
-            ).build();
-
-        ClusterState currentState = clusterService.state();
-        PlannerContext plannerContext = e.getPlannerContext(currentState);
-        BoundAddColumn addXLong = AlterTableAddColumnPlan.bind(
-            e.analyze("alter table t add column x long"),
-            plannerContext.transactionContext(),
-            plannerContext.nodeContext(),
-            Row.EMPTY,
-            SubQueryResults.EMPTY,
-            null
-        );
```

Comment:
Removing tests here since new indexer branch already uses new `TransportAddColumnAction`.
We still have one [usage](https://github.com/crate/crate/blob/master/server/src/main/java/io/crate/execution/dml/TransportShardAction.java#L96) of the `TransportSchemaUpdateAction` in the parent class but it should be removed once we migrate fully to the indexer - so `TransportSchemaUpdateAction` will be removed in the future as well.